verification/curator-service/ui/src/components/App.tsx

```diff
@@ -121,6 +121,12 @@ const useStyles = makeStyles((theme: Theme) => ({
     root: {
         display: 'flex',
     },
+    buttonLabel: {
```

Comment:
Hm -- should we apply this globally to .button in `index.css`?

Reply:
That's a good idea but it's a little awkward to do that. First it has to be applied to the class `.MuiButton-label` which while I'm sure it won't change anytime soon, that name is controlled by the material code and therefore can change. Secondly, the styling from the material code supersedes the logic from `index.css`, so we would have to double up the selector making the code in `index.css` the following:
```css
.MuiButton-label.MuiButton-label {
  display: block;
  [...]
}
```

On top of that, right now we don't actually have anymore buttons that can be pushed to have a small width. Due to all of that I'll keep as is for now.